atlp-rwanda/code-ninjas-be
# #181414759 Ft(login with social media) added google and Facebook login functionality

### What does this PR do?
 Login with google and facebook
### Description of Task to be completed?
Currently a new user cannot login with their pre-existing social media accounts. He/She can only login with email password combinations.

Now user  should is able to login via Google and Facebook

### How should this be manually tested?
Get started By Running commands bellow:
  - clone repo
  - switch to [ft-login-with-google-and-facebook-181414759](https://github.com/atlp-rwanda/code-ninjas-be/tree/ft-login-with-google-and-facebook-181414759) branch
  - ``yarn install``: to install all dependencies
  - Create .env file
  - Set up .env as stated in .env example
  - run `yarn run  dev` to start the server
  - Open your browser and open on this route `/api/auth/login` and choose between Facebook and google
  - You can also login with `facebook` directly using this route `/api/auth/facebook`
  - -You can also login with `google` directly using this route `/api/auth/google`
  - After successful logging in  you will be able to see this message `{"Message":"logged in successfully"}`

### Any background context you want to provide?
### What are the relevant pivotal tracker stories?
[#181414759](https://www.pivotaltracker.com/story/show/181414759)
